molgenis-core-ui/src/main/java/org/molgenis/core/ui/style/StyleSheet.java

```diff
@@ -1,54 +0,0 @@
-package org.molgenis.core.ui.style;
-
-import org.molgenis.data.Entity;
-import org.molgenis.data.file.model.FileMeta;
-import org.molgenis.data.meta.model.EntityType;
-import org.molgenis.data.support.StaticEntity;
-
-@SuppressWarnings("unused")
-public class StyleSheet extends StaticEntity {
```

Comment:
@fdlk do we need a migration step to remove the table form the db ?

Reply:
No, that happens automagically